pkg/build/api/types.go

```diff
@@ -425,6 +428,19 @@ type SourceBuildStrategy struct {
 	ForcePull bool
 }
 
+// JenkinsPipelineStrategy holds parameters specific to a Jenkins Pipeline build.
+type JenkinsPipelineBuildStrategy struct {
```

Comment:
I think i'd prefer to keep "traditional" jenkins jobs separate from "pipeline" jobs, but i'm not sure if that's a valid bias or not.